### PR TITLE
Drop support for node 8

### PR DESCRIPTION
**Pull request by gilest**
Re: #306 

This is all I can think of doing.

Let's see if the tests pass 🤷‍♀

### CODE REVIEW

**Comment by gilest** [comment]
Have added Compatability section. The `ember-cli` version I chose is just the version from `package.json` not sure what we actually support.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ Uploads continue in the background, even after a page transition. In other words
 
 [View the docs here.](https://adopted-ember-addons.github.io/ember-file-upload/docs/)
 
+## Compatibility
+
+Ember.js v2.18 or above
+Ember CLI v3.5.1 or above
```
Comment (jelhan) [inline README.md]: > The ember-cli version I chose is just the version from package.json not sure what we actually support.

As we don't do anything special in the build pipeline of this addon, going with defaults of Ember CLI blueprint should be fine:

```suggestion
Ember CLI v2.13 or above
```

Comment (gilest) [reply README.md]: 👍

**Comment by gilest** [comment]
@jelhan you may merge when ready.

**Pull request merged**